app/assets/images/blacklight/maps.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
+  <image id="Ellipse_1" data-name="Ellipse 1" x="5" y="1" width="14" height="22" xlink:href="data:img/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAWCAYAAADwza0nAAAA/klEQVQ4jZWSMU4CQRSGP8UOGw9gVilMMJEzmG2x1EjhESy9AOEAWJBwAyhI6Gih4gJgZcUNSAg1mIE3yfDeW4Ev+beY//9n3+zOBZYHoA48iTMHRsCvSSZkQAdYAFvRQtYykxbegU1S0NpI5oDcCRYpj8VLYF0Q8rSWDh+OGTQVeV7o0HOMVnKOluP3gzFxDI32x2HWpYnt/2PkxbiwLAFV4FkZDfkAYb0L6IsyCI+KM8ox3ccdvs8otfXcKyektTKnBV6doNabaQkzJxz1Y9IJj04hqmbSir5TGpqUQ9kp3pxSDHwlpeappUjRvd1x9U/xE7g+922BW+DOdYA/IUiSrESa4EIAAAAASUVORK5CYII="/>
```

Comment:
We might want to remove the `id` and `data-name`. Is it possible to not have embedded images and use svg shapes instead? Adopters may want to customize the colors downstream.